src/Symfony/Component/Debug/Tests/Fixtures/DeprecatedClass.php

```diff
@@ -4,7 +4,7 @@
 
 /**
  * @deprecated but this is a test
- *             deprecation notice.
+ *             deprecation notice
```

Comment:
corresponding assertion needs to be updated

Reply:
https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Debug/Tests/DebugClassLoaderTest.php#L141

Reply:
Thanks